hack/deploy-cluster.sh

```diff
@@ -74,4 +74,40 @@ aks-engine deploy --subscription-id ${SUBSCRIPTION_ID} \
   --client-id ${CLIENT_ID} \
   --client-secret ${CLIENT_SECRET}
 echo "Kubernetes cluster deployed. Please find the kubeconfig for it in _output/"
+export KUBECONFIG=_output/kubeconfig/kubeconfig.$LOCATION.json
 
+#Deploy AzureDisk CSI Plugin
+COMMAND_DISK="kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/deploy"
+config_disk=("crd-csi-driver-registry.yaml" "crd-csi-node-info.yaml" "rbac-csi-attacher.yaml" "rbac-csi-driver-registrar.yaml" "rbac-csi-provisioner.yaml" "rbac-csi-snapshotter.yaml" "csi-azuredisk-provisioner.yaml" "csi-azuredisk-attacher.yaml" "azuredisk-csi-driver.yaml" "example/storageclass-azuredisk-csi.yaml")
+for i in {1..10}
+do
+$COMMAND_DISK/$config_disk
+done
+#Deploy AzureFile CSI Plugin
+COMMAND_FILE="kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy"
+config_files=("crd-csi-driver-registry.yaml" "crd-csi-node-info.yaml" "rbac-csi-attacher.yaml" "rbac-csi-driver-registrar.yaml" "rbac-csi-provisioner.yaml" "rbac-csi-snapshotter.yaml" "csi-azurefile-provisioner.yaml" "csi-azurefile-attacher.yaml" "azurefile-csi-driver.yaml" "example/storageclass-azurefile-csi.yaml")
+for i in {1..10}
+do
+$COMMAND_FILE/$config_files
+done
+echo "Do you want to change the default storage class?[Yes/No]"
```

Comment:
Could we change this into an environment variable, e.g. `USE_CSI_DEFAULT_STORAGECLASS`? your can refer L30 for checking whether it is set or not